src/smk/VertexArray.cpp

```diff
@@ -40,7 +40,7 @@ VertexArray::VertexArray(VertexArray&& other) noexcept {
 }
 
 VertexArray& VertexArray::operator=(const VertexArray& other) {
-  this->~VertexArray();
+  this->Release();
```

Comment:
Can this be directly: `Released()` without the 'this->`
?

Reply:
It sure can be. Just my habit to mention `this` and `other` when overloading operators sorry.